.github/workflows/mirror.yml

```diff
@@ -0,0 +1,22 @@
+name: Mirror Repo
+
+on:
+  # Runs on pushes targeting the default branch
+  push:
+    branches: ["main"]
+
+jobs:
+  mirror:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - uses: yesolutions/mirror-action@v0.7.0
```

Comment:
In the future, we could probably simplify this down to a shell script, though I think later @karras will be suggesting policies around third-party GHAction usage.